sounddevice.py

```diff
@@ -2427,6 +2427,10 @@ def __init__(self, exclusive=False):
             Exclusive mode allows to deliver audio data directly to
             hardware bypassing software mixing.
 
+        auto_convert : bool
+            auto_convert allows to resample audio data to hardware
+            supported samplerate when in shared mode(None exclusive mode)
```

Comment:
According to https://portaudio.com/docs/v19-doxydocs/pa__win__wasapi_8h_source.html, this also allows mixing, not only resampling:

> allow API to insert system-level channel matrix mixer and sample rate converter to allow
playback formats that do not match the current configured system settings.
this is in particular required for streams not matching the system mixer sample rate.
only applies in Shared mode.

Reply:
I'm updating this text in #494.